docs/cluster-management/config.md

```diff
@@ -72,6 +72,9 @@ tags:  # <string>: <string> map of key/value pairs
 # whether to use spot instances in the cluster (default: false)
 # see https://cortex.dev/v/master/cluster-management/spot-instances for additional details on spot configuration
 spot: false
+
+# ARN of SSL certificate in AWS Certificate Manager (ACM) for HTTPS (default: null)
+ssl_certificate_arn: # if empty, APIs will only be accessible by HTTP
```

Comment:
Could you revisit https://www.cortex.dev/miscellaneous/security#https? I can too, let me know if you prefer me to